linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -124,23 +126,14 @@
             database,
             network,
             num_other_initial_chains: 10,
-            initial_amount: Amount::from_tokens(10),
+            initial_amount: Amount::from_tokens(1_000_000),
+            policy: ResourceControlPolicy::devnet(),
```

Comment:
I'm interested in what could cause this lint error once you find it out 😅🤔

Reply:
`linera-service`'s `test` feature didn't depend on `linera-execution/test`. I added a fix.